src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -487,6 +487,7 @@ public net.minecraft.world.level.block.FireBlock getIgniteOdds(Lnet/minecraft/wo
 public net.minecraft.world.level.block.PointedDripstoneBlock LAVA_TRANSFER_PROBABILITY_PER_RANDOM_TICK # LAVA_TRANSFER_PROBABILITY_PER_RANDOM_TICK
 public net.minecraft.world.level.block.PointedDripstoneBlock WATER_TRANSFER_PROBABILITY_PER_RANDOM_TICK # WATER_TRANSFER_PROBABILITY_PER_RANDOM_TICK
 public net.minecraft.world.level.block.entity.BlockEntityType$BlockEntitySupplier
+private-f net.minecraft.world.level.block.entity.BlockEntityType validBlocks
```

Comment:
A possible alternative is to make the set mutable instead.

Reply:
Orion was heavily against that originally which is why I was making this event as a compromise. Unless we override Orion and just AT the field

Reply:
Orion is completely right. Priority #1 is that we must ensure that modders only modify the set of valid blocks from within the event.

Reply:
The -f here is so the event can set the field to the event's final set. The event's set is already mutable